### PR TITLE
DX: ProjectCodeTest - fix detection by testExpectedInputOrder

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

File: tests/Fixer/FunctionNotation/NullableTypeDeclarationForDefaultNullValueFixerTest.php [hunk 1]
```diff
@@ -359,55 +365,60 @@ public function testFixInversePhp74(string $expected, string $input): void
...
-    public function testFix80(string $input, ?string $expected = null): void
+    public function testFix80(string $expected, ?string $input = null): void
     {
-        if (null === $expected) {
-            $this->doTest($input);
+        if (null === $input) {
+            $this->doTest($expected);
         } else {
             $this->doTest($expected, $input);
         }
```
Comment (kubawerlos) [inline tests/Fixer/FunctionNotation/NullableTypeDeclarationForDefaultNullValueFixerTest.php:417]:
```suggestion
        $this->doTest($expected, $input);
```

**Review by kubawerlos — CHANGES_REQUESTED** [review]

**Comment by keradus** [comment]
@fabpot , fabbot.io is wrong here, it assumes it runs on Symfony. maybe it could run rules depending on repos (default rules or default+Symfony rules?)

**Review by kubawerlos — APPROVED** [review]

**Comment by GrahamCampbell** [comment]
Merge conflicts. 😭

**Comment by keradus** [comment]
not sure what this comment brings to discussion, each of us can see it, no need to fingerprint it :shrug:

**Pull request merged**